ci_framework/roles/artifacts/tasks/cluster_info.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Check for oc command
```

Comment:
maybe using/consuming the `{{ cifmw_path }}` param would help there. There are examples of it already, and it's created directly in the ci_setup if no one sets it in the environment.
That would ensure ~/.crc/bin/oc is in the PATH, for instance.